### PR TITLE
Expanded documentation for aes-colour-fill-alpha

### PR DESCRIPTION
**Pull request by kelseymoty**
Expanded documentation for aes-colour-fill-alpha addresses one of the points raised in issue #3297

### CODE REVIEW

**Comment by clauswilke** [comment]
See also #3788.

File: R/aes-colour-fill-alpha.r [hunk 1]
```diff
@@ -1,7 +1,48 @@
 #' Colour related aesthetics: colour, fill and alpha
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: I think there should be a comma after "fill".

File: R/aes-colour-fill-alpha.r [hunk 2]
```diff
@@ -1,7 +1,48 @@
...
 #'
-#' This page demonstrates the usage of a sub-group
-#' of aesthetics: colour, fill and alpha.
+#' These aesthetics parameters change the colour (`colour` and `fill`) and the
+#' opacity (`alpha`) of geom elements on a plot. Almost every geom has either
+#' colour or fill (or both), as well as can have their alpha modified.
+#' Modifying colour on a plot is useful way to enhance the presentation of data,
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: is *a* useful way

File: R/aes-colour-fill-alpha.r [hunk 3]
```diff
@@ -1,7 +1,48 @@
...
+#' often especially when a plot graphs more than two variables.
+#'
+#' @section Colour and fill:
+#'
+#' Colours and fills can be specified in the following ways:
+#' * A name, e.g., "red". R has 657 built-in named colours, which can be
+#' listed with colours().
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: Put `colours()` in back quotes, and maybe also in square brackets to create a link. Please try if this works before committing this change.

File: R/aes-colour-fill-alpha.r [hunk 4]
```diff
@@ -1,7 +1,48 @@
...
+#' * An rgb specification, with a string of the form "#RRGGBB" where each of the
+#' pairs RR, GG, BB consists of two hexadecimal digits giving a value in the
+#' range 00 to FF. You can optionally make the colour transparent by using the
+#' form "#RRGGBBAA".
+#' * An NA, for a completely transparent colour.
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: Put `NA` in back quotes.

File: R/aes-colour-fill-alpha.r [hunk 5]
```diff
@@ -1,7 +1,48 @@
...
+#'
+#' @section Alpha:
+#'
+#' Alpha refers to the opacity of a geom. Values of alpha range from 0 to 1,
+#' with lower values reflecting more transparent objects.
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: I'm not sure I like "reflecting". Maybe write simply: "with lower values corresponding to more transparent colors".

File: R/aes-colour-fill-alpha.r [hunk 6]
```diff
@@ -1,7 +1,48 @@
...
+#'
+#' Alpha can additionally be modified through the `colour` or `fill` if either
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: Do you mean: through the `colour` or `fill` *aesthetic*?

File: R/aes-colour-fill-alpha.r [hunk 7]
```diff
@@ -1,7 +1,48 @@
...
+#' aesthetic is modified using an rgb specification ("#RRGGBBAA"), where "A"
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: Further up you wrote RR, GG, BB (no quotes, double letters). Similarly, here I think you should write "where AA..."

File: R/aes-colour-fill-alpha.r [hunk 8]
```diff
@@ -1,7 +1,48 @@
...
+#' refers to transparency values.
+#'
+#'
+#' @seealso * Other options for modifying colour:
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: Does this bullet-pointed list render correctly in the .Rd file?

**Comment by thomasp85** [comment]
@kelseymoty, are you interested in finishing this off?

**Comment by clauswilke** [comment]
Thanks, this will be included in the next ggplot2 release.

**Pull request merged**